package.json

```diff
@@ -4,6 +4,7 @@
   "description": "npm's markdown parser",
   "main": "index.js",
   "scripts": {
+    "prebuild": "node prebuild.js",
```

Comment:
Hmm, speaking of which, I suppose it would be helpful to rebuild the `marky.json` in a `postversion` hook too. Lemme do that.